doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -22,7 +22,7 @@ The most relevant changes that are present on the main branch of the |NCS|, as c
 Highlights
 **********
 
-* |no_changes_yet_note|
+* Removed support for Pelion DM library.
```

Comment:
```suggestion
* Removed support for the Pelion DM library.
```

Reply:
Quick question - can we fix it when release notes are moved? I am wondering if this justifies the CI rerun.

Reply:
Approving the PR, considering the Pelion will be removed from the known Issue in a separate ticket.

Reply:
I was rather thinking about leaving it as is (so we can use green CI) and add the missing `the` when there is a move from changelog to 1.8 release notes. There is always a review done at that moment where all such things are fixed. If you don't mind let's merge it.

Reply:
merged